src/views/AvailableTeams.vue

```diff
@@ -29,21 +29,34 @@
         </p>
       </router-link>
       <b-button class="create" @click="createTeam">Create New Team</b-button>
+      <b-button v-if="userData.privilegeLevel === userConstants.ADMIN"
+                class="create"
+                @click="downloadCSV">
+          Download Teams CSV
+      </b-button>
   </div>
 </template>
 
 <script>
 import Vue from 'vue';
 import VueRouter from 'vue-router';
 import { mapState } from 'vuex';
+import { getTeamsCSV } from '../api/api';
+import userConstants from '../constants/userConstants';
```

Comment:
So you can change this import to from '../auth/constants'

Reply:
Fixed, see 877e07032f295e2ca1f3cd244d365807356b4b5a.